vuepress/docs/next/tutorials/devops/change-default-datasource.md

```diff
@@ -77,77 +56,77 @@ container.
-repeat this exercise for both the `eap` and `wildfly` profiles. 
-
-### 5. Build and push your custom Docker image
-
-Before building your Docker image, it would be a good idea to change the name of the image you want to build. Navigate
-to the previously modified `<image>` element in the Fabric8 Maven Docker Plugin.You can change the `<name>` element
-to reflect your preferred name, e.g.
-
-`<name>your-docker-registry.com/your-org/your-entando-app:1.0.0</name>`
+### Build and push your custom Docker image
 
-Run:
-
-`mvn clean package -Pwildfly -Pderby`
-
-A new Docker image should now be available named `your-docker-registry.com/your-org/your-entando-app:1.0.0`
+1. Before building your Docker image, it is a good idea to change the name of the image you want to build. Navigate
+to the previously modified `<image>` element in the Fabric8 Maven Docker Plugin. You can change the `<name>` element
+to reflect your preferred name:
+e.g.,  
+``` bash
+<name>YOUR-DOCKER-REGISTRY.com/YOUR-ORG/YOUR-ENTANDO-APP:1.0.0</name>
+```
+**Does this work for the tomcat version?**
+2. Run:
+``` bash
+mvn clean package -Ptomcat -Pderby
+```
 
-Log into the Docker registry in question and then push the image:
+A new Docker image should now be available at your Docker registry.
 
-`docker push your-docker-registry.com/your-org/your-entando-app:1.0.0`
+3. Log into the Docker registry in question and then push the image:
 
+``` bash
+docker push YOUR-DOCKER-REGISTRY.com/YOUR-ORG/YOUR-ENTANDO-APP:1.0.0`
+```
 You are now ready to deploy this image.
 
-### 6. Deploy your EntandoApp with the correct environment variables
+**Should we change the following steps with list of recommended env variables instead?**
+### Deploy your EntandoApp with the correct environment variables
  
 The final step is to configure your EntandoApp deployment with the correct environment variables. As is the case with
-all the Entando Custom Resources that result in actual deployments, the property `spec.environmentVariables` will be translated
-into environment variables on each of the Containers in the Deployment's Pod. For an EntandoApp named 'my-app',
-the new state of the EntandoApp would could be placed in a file named `my-app.yaml` that would look something like this: 
+all the Entando Custom Resources that result in actual deployments, the property `spec.environmentVariables` will be translated into environment variables on each of the containers in the deployment's pod. 
+
+e.g., for an EntandoApp named `quickstart`: 
 ```
       kind: "EntandoApp"
       metadata:
-        name: "my-app"
+        name: "quickstart"
       spec:
         dbms: postgresql
         replicas: 1
-        customServerImage: your-docker-registry.com/your-org/your-entando-app:1.0.0
-        ingressPath: /your-entando-app
+        customServerImage: YOUR-DOCKER-REGISTRY.com/YOUR-ORG/YOUR-ENTANDO-APP:1.0.0
+        ingressPath: /YOUR-ENTANDO-APP
```

Comment:
@nshaw @pietrangelo for this section, do we check env variables against server.xml or another file? And should we give a list of recommended env variables or just provide sample?

Reply:
@pietrangelo One idea @jyunmitch and I had discussed was creating a reference file in the engine repo to contain the list of supported env variables, and then we could point to it from the docs. That would make it easier for the dev team to keep the list in sync with the implementation. CC: @eugeniosant